tests/js/call-missing-first-arg/jsfmt.spec.js

```diff
@@ -0,0 +1 @@
+run_spec(__dirname, ["babel-flow", "babel-ts"]);
```

Comment:
When I tried with "babel" I did encounter what I think was a parse error. I think it would be ideal if we could test with `["babel", "babel-ts"]` or event `["babel", "typescript"]`.